### PR TITLE
Add 001,002/inohiro

### PR DESCRIPTION
**Pull request by inohiro**
Add 001,002/inohiro

### CODE REVIEW

File: 001/inohiro/001.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+select
+    sum(s.i)
+from (
+    select
+        *
+    from
+        generate_series(0, 1000-1)
```
Comment (k-mats) [inline 001/inohiro/001.sql:7]: postgres?

**Comment by k-mats** [comment]
:eyes:

**Comment by k4zy** [comment]
:eyes:

File: 001/inohiro/001.sql (hunk 1 above)
Comment (inohiro) [inline 001/inohiro/001.sql:7]: Only PostgreSQL :-1:

**Pull request merged**